dp/generate_climos.py

```diff
@@ -134,19 +134,34 @@ def climo_outputs(time_resolution):
     # Apply per-variable processing
     # - Scale pr variable if it is not in desired units
     #   - TODO: Use UDUNITS here for more robust processing of units? It has a fairly heavy API. But this is uuuugly.
+    pr_attributes = {}  # will contain updates, if any, to pr variable attributes
     if 'pr' in input_file.dependent_varnames:
```

Comment:
I know that you didn't create this problem, but I don't like the big, long branch here for `pr` specific code. Could we stash that away in a subroutine or something to help this read better for the common cases?